internal/command/views/hook_ui.go

```diff
@@ -525,7 +525,39 @@ func (h *UiHook) PostListQuery(id terraform.HookResourceIdentity, results plans.
 			value.GetAttr("display_name").AsString(),
 		))
 	}
+	return terraform.HookActionContinue, nil
+}
+
+func (h *UiHook) StartAction(id terraform.HookActionIdentity) (terraform.HookAction, error) {
+	h.println(fmt.Sprintf(
+		h.view.colorize.Color("[reset][bold]Action started: %s[reset]"),
+		id.String(),
+	))
+	return terraform.HookActionContinue, nil
+}
+
+func (h *UiHook) ProgressAction(id terraform.HookActionIdentity, progress string) (terraform.HookAction, error) {
```

Comment:
Should we give actions a progress bar like resource have?

Reply:
How would that work, wouldn't the action need to report a percentage done back then?